fe/fe-core/src/test/java/com/starrocks/sql/plan/ArrayTypeTest.java

```diff
@@ -186,4 +188,21 @@ public void testArrayClone() throws Exception {
                 "  |  <slot 8> : array_contains(ARRAY<bigint(20)>[7: expr], 1)\n" +
                 "  |  <slot 9> : array_contains(ARRAY<bigint(20)>[7: expr], 2)"));
     }
+
+    @Test
+    public void testArrayWindowFunction() {
+        for (String fnName : Sets.newHashSet(AnalyticExpr.LASTVALUE, AnalyticExpr.FIRSTVALUE)) {
+            try {
+                String sql = String.format("select %s(v3) over() from tarray", fnName.toLowerCase());
+                getFragmentPlan(sql);
+            } catch (Exception e) {
```

Comment:
please use ExpectedException.

you could refer to ColocateTableTest

Reply:
Indeed, but here is the thing that I want to test multiply functions in one test